code/modules/mob/living/simple_animal/hostile/boss/boss.dm

```diff
@@ -0,0 +1,8 @@
+// Only here to serve as a filter, say you don't want boss mobs to spawn from gold slimes
```

Comment:
Ex_act will send a return value for the timelost spellcasters. Sentience potion was an oversight :P

They're meant for gateway missions, so SM is not really a concern. Wand of Death is though.